onBoardProject/screens/add-user-screen.tsx

```diff
@@ -40,13 +56,21 @@ export const AddUserScreen: React.FC = props => {
       validPassword &&
       validRole
     ) {
-      Navigation.pop(props.componentId);
+      createUser({
+        variables: {
+          name: name,
+          email: email,
+          phone: phone,
+          birthDate: birthDate,
+          password: password,
+          role: role,
```

Comment:
Quando o campo tem o mesmo nome da variável q vc tá enviando, vc pode omitir:

```suggestion
          name,
          email,
          phone,
          birthDate,
          password,
          role,
```